bin/pycbc_compress_bank

```diff
@@ -40,72 +40,105 @@ from pycbc import filter
 
 parser = argparse.ArgumentParser(description=__description__)
 parser.add_argument("--bank-file", type=str,
-                help="Bank hdf file to load.")
+                    help="Bank hdf file to load.")
 parser.add_argument("--output", type=str,
-                help="The hdf file to save the templates and compressed "
-                     "waveforms to.")
+                    help="The hdf file to save the templates and compressed "
+                    "waveforms to.")
 parser.add_argument("--low-frequency-cutoff", type=float, required=True,
-                help="The low frequency cutoff to use for generating "
-                     "the waveforms (Hz)")
+                    help="The low frequency cutoff to use for generating "
+                    "the waveforms (Hz)")
 # add approximant arg
 pycbc.waveform.bank.add_approximant_arg(parser)
 parser.add_argument("--sample-rate", type=int, required=True,
-                help="Half this value sets the maximum frequency of the "
-                     "compressed waveforms.")
+                    help="Half this value sets the maximum frequency of the "
+                    "compressed waveforms.")
 parser.add_argument("--tmplt-index", nargs=2, type=int, default=None,
-                help="Only generate compressed waveforms for the given "
-                     "indices in the bank file. Must provide both a start "
-                     "and a stop index. Default is to compress all of the "
-                     "templates in the bank file.")
-parser.add_argument("--compression-algorithm", type=str, required=True,
-                choices=compress.compression_algorithms.keys(),
-                help="The compression algorithm to use for selecting "
-                     "frequency points.")
+                    help="Only generate compressed waveforms for the given "
+                    "indices in the bank file. Must provide both a start "
+                    "and a stop index. Default is to compress all of the "
+                    "templates in the bank file.")
+parser.add_argument("--compression-algorithm", nargs='*', type=str,
+                    required=True,
+                    choices=compress.compression_algorithms.keys(),
+                    help="Specify the waveform model and compression"
+                    "algorithm to use for selecting frequency points"
+                    "for TaylorF2 templates and SEOBNRv2_ROM templates."
+                    "This would be used to select sets of frequency"
+                    "points for amplitude and phase interpolants. Either"
+                    "specify different algorithms for the TaylorF2 and"
+                    "SEOBNRv2_ROM or specify a common compression"
+                    "algorithm compatible with both. For example"
+                    "partial_rom_compression works only for the ROMs."
+                    "Look at compress.compression_algorithms.keys()"
+                    "for the available options")
```

Comment:
Don't make the combination of approximant and algorithms possible choices. Instead, just list here the possible algorithms, and make it so that an approximant name can be added to only apply that algorithm to templates with that approximant. Also don't limit the number of algorithms that may be used to 2. If at some point we end up using more than two approximants in a bank (as we nearly did for O2), you'll want the code to be able to support that. I suggest something like:
```
parser.add_argument("--compression-algorithm", nargs='+', type=str,
                    required=True, metavar="ALGORITHM[:APPRX]",
                    help="Specify the compression algorithm to use. Choices "
                         "are {}. ".format(
                         ', '.join(compress.compression_algorithms.keys())) +
                         "To only use an algorithm with a specific "
                         "approximant, add the approximant name preceeded by "
                         "a colon; e.g., 'spa:TaylorF2'. In this case, an "
                         "algorithm must be provided for every approximant "
                         "that exists in the bank.")
```